### PR TITLE
Added splunkmint action

### PR DESCRIPTION
**Pull request by xfreebird**
This adds option to upload the app.dSYM.zip file to Splunk MINT through their Rest API.

The spec errors from this branch are coming from the master branch.

### CODE REVIEW

File: lib/fastlane/actions/splunkmint.rb [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :proxy_password,
+                                       env_name: "FL_SPLUNKMINT_PROXY_PASSWORD",
+                                       description: "Proxy password",
+                                       default_value: nil,
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :proxy_address,
+                                       env_name: "FL_SPLUNKMINT_PROXY_ADDRESS",
+                                       description: "Proxy address",
+                                       default_value: nil,
```
Comment (KrauseFx) [inline lib/fastlane/actions/splunkmint.rb]: You can remove the `default_value: nil` as it will be `nil` by default anywa

**Comment by KrauseFx** [comment]
This looks great, thanks for the pull request.

**Comment by KrauseFx** [comment]
I'll change the default_value

**Comment by KrauseFx** [comment]
Thanks for your contribution :rocket:

File: lib/fastlane/actions/splunkmint.rb (hunk 1 above)
Comment (xfreebird) [inline lib/fastlane/actions/splunkmint.rb]: Thanks! :+1:

**Comment by KrauseFx** [comment]
@xfreebird is this iOS and Mac only?

**Comment by xfreebird** [comment]
iOS only

**Comment by KrauseFx** [comment]
Thanks, I updated the platform definition.

**Comment by xfreebird** [comment]
It seems there is a rubocop issue"lib/fastlane/actions/splunkmint.rb:75:1: C: Trailing whitespace detected." it is failing now the Travis build

**Comment by KrauseFx** [comment]
Thanks - already pushed a fix

**Comment by xfreebird** [comment]
:+1:

**Comment by Koani** [comment]
Awesome *_*
No need to include that build script in all my targets. Thank you!

**Pull request merged**